diesel_cli/src/print_schema.rs

```diff
@@ -275,7 +279,14 @@ impl Display for CustomTypeList {
                         writeln!(out, "///")?;
                         writeln!(out, "/// (Automatically generated by Diesel.)")?;
                     }
-                    writeln!(out, "#[derive(diesel::sql_types::SqlType)]")?;
+                    if self.derives.is_empty() {
+                        writeln!(
+                            out,
+                            "#[derive(diesel::sql_types::SqlType, diesel::query_builder::QueryId)]"
+                        )?;
+                    } else {
+                        writeln!(out, "#[derive({})]", self.derives.join(", "))?;
```

Comment:
I think it would be reasonable to always put the `SqlType` derive by default. I can't think of a scenario where people would want to not put it.